tools/cross/format.py

```diff
@@ -97,9 +97,13 @@ def clang_format(files: List[str], check: bool = False) -> bool:
 def prettier(files: List[str], check: bool = False) -> bool:
     cmd = [PRETTIER]
     if check:
-        cmd.append("--check")
+        # This flag will only print files that require formatting
+        # and change exit code of the process depending on state.
+        cmd.append("--list-different")
     else:
         cmd.append("--write")
+        # This will only print files that requires formatting.
+        cmd.append("--log-level=warn")
```

Comment:
```suggestion
    cmd.append("--log-level=warn")
```

But this is great so it doesn't print when there's nothing wrong